mod/01_rating/set/type/metric_value.rb

```diff
@@ -61,18 +63,24 @@ def clone_subcards_to_hash subcards
   source_subcards
 end
 
+def get_source_card sub_source_card
+  Env.params[:sourcebox] = 'true'
+  card =
+    if (new_source_card = sub_source_card.subcard('new_source'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.